docs/OpenShift GitOps Usage Guide.md

```diff
@@ -0,0 +1,1291 @@
+
+2. Make sure *ArgoCDAdmins *group has required permissions in the argocd-rbac configmap. Please refer to the ArgoCD Rbac documentation [here](https://argoproj.github.io/argo-cd/operator-manual/rbac/).
+
+In this case, I wish to provide admin access to the users in *ArgoCDAdmins *groupargocd-rbac-cmArgoCDAdmins
+
+`$ kubectl edit configmap argocd-rbac-cm -n openshift-gitops`
+
+```
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: argocd-rbac-cm
+data:
+  policy.csv: |
+    g, ArgoCDAdmins, role:admin
+```
+
+### Working with Dex
+
+**Note: **For a fresh install of v1.3.0, Dex is automatically configured. You can log into the default Argo CD instance in the openshift-gitops namespace using the OpenShift or kubeadmin credentials. As an admin you can disable the Dex installation after the Operator is installed which will remove the Dex deployment from the openshift-gitops namespace.
+
+
+
+**For upgrades, **We can enable it by updating the Subscription resource for the OpenShift Gitops Operator.
+
+```
+spec:
+  config:
+    env:
+    - name: DISABLE_DEX
+      value: "false"
+```
+
+`oc patch subscriptions.operators.coreos.com/openshift-gitops-operator -n openshift-operators --type='merge' --patch '{ "spec": { "config": { "env": [ { "name": "DISABLE_DEX", "value": "false" } ] } } }'`
+
+This will get the argocd-cluster-dex-server instance running. To enable login with openshift, you need to update the argo-cd CR with the below field in spec. 
+
+```
+spec:
+  dex:
+    openShiftOAuth: true
+```
+
+`oc patch argocd/openshift-gitops -n openshift-gitops --type='merge' --patch='{ "spec": { "dex": { "openShiftOAuth": true } } }'`
+
+You can enable RBAC on argo-cd by following the instructions provided in the argo-cd [RBAC Configuration](https://argoproj.github.io/argo-cd/operator-manual/rbac/). Example RBAC configuration looks like this.
+
+```
+spec:
+  dex:
+    openShiftOAuth: true
+  rbac:
+    defaultPolicy: 'role:readonly'
+    policy: |
+      g, system:cluster-admins, role:admin
+      g, ocp-admins, role:admin
+    scopes: '[groups]'
+```
+
+
+`oc patch argocd/openshift-gitops -n openshift-gitops --type='merge' --patch='{ "spec": { "rbac": { "defaultPolicy": "role:readonly", "scopes": "[groups]", "policy": "g, system:cluster-admins, role:admin\ng, ocp-admins, role:admin" } } }'`
+
+
+#### Restricting dex / openShiftOAuth to only a set of groups
+
+As discussed here [https://cloud.redhat.com/blog/openshift-authentication-integration-with-argocd](https://cloud.redhat.com/blog/openshift-authentication-integration-with-argocd) you can restrict oauth access to certain groups. Currently it is not possible to restrict the ArgoCD to only a set of groups, through `openShiftOAuth: true`, the RFE is tracked upstream [here](https://github.com/argoproj-labs/argocd-operator/issues/391). However, you can let the operator generate the oauth client and dex.config and then configure it manually and thus be able to extend it:
```

Comment:
the issue here is closed? So is this point no longer needed?

Reply:
also looks like something followed it when nothing does now, remove the :

Reply:
I'm not too sure about your first question, leaving it as is for now since thats how its captured in the usage guide

Reply:
yeah no worries, maybe something that just need to be updated in both.

Reply:
I know that the groups thing works now with dex, but i wasn't sure how that affects the documentation that follows and whether that was originally meant to be a workaround ,and is not needed now or something else